### ISSUE TITLE
Items received via mail will not Disenchant -or- open in the case of boxes

### ISSUE THREAD

**Issue by AyleeJenn** [slowtorta/turtlewow-bug-tracker#279]
**Please, provide a clear description what the bug is:**
Any items received via mail cannot be disenchanted or "opened" (like the turtle mode reward boxes)

**Steps to reproduce the behavior with as much detail as possible:**

1.  Obtain any lock box or items vial in-game mail
2.  Try disenchanting or opening them
3.  Observe as nothing happens

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**
Any item sent via mail appears to be affected

NPC: https://database.turtle-wow.org/?npc= N/A
Item: https://database.turtle-wow.org/?item=N/A
Quest: https://database.turtle-wow.org/?quest= N/A
Spell: https://database.turtle-wow.org/?spell= N/A
Object: https://database.turtle-wow.org/?object= N/A

**Expected behavior. Describe how it should work:**
A loot window should open with disenchanting materials, when trying to disenchant the items
A loot window should open with boxes that are opened (like the turtle mode reward boxes)

**Comment by sonpansatan** [comment]
I am also not able to disenchant anything (Lalarularu).

**Comment by AyleeJenn** [comment]
A GM just handled my items and now I could disenchant them.

Previously, before the GM intervention, I tried giving these items to another player to see if they could disenchant them. The other player could also not disenchant them. I am a bit puzzled by this but thought I would report it and update with the progress

**Comment by AyleeJenn** [comment]
Update: I did some research on this and appears to be a mail related problem.

How I arrived at this conclusion:
A GM graciously fixed one of my characters who had this bug. But then later that day more items were sent to me for disenchanting. When I tried disenchanting, after the GM had helped my character ... I was unable to disenchant things again that were sent via mail to me.

On a different character I tried opening the speedy level 20 turtle mode box and was unable to. Again, this box is sent via -mail- system.

Therefore, I feel that maybe the mailing system is, somehow, involved in this bug as anything sent via mail does not seem to allow for loot window to open at the moment.
